.github/workflows/publish.yml

```
@@ -2,70 +2,160 @@
+
+                  git config user.name "github-actions[bot]"
+                  git config user.email "github-actions[bot]@users.noreply.github.com"
+                  git tag -a "$TAG" -m "Release $TAG"
+                  git push origin "$TAG"
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    trigger-agent-manager:
+        needs: build
+        if: github.ref_name == 'dev'
+        uses: ./.github/workflows/trigger-workflow.yml@dev
```

actionlint (1.7.8)

reusable workflow call "./.github/workflows/trigger-workflow.yml@dev" at "uses" is not following the format "owner/repo/path/to/workflow.yml@ref" nor "./path/to/workflow.yml". see https://docs.github.com/en/actions/learn-github-actions/reusing-workflows for more details

(workflow-call)